### PR TITLE
`not` instead of `no` - typo fix

### PR DESCRIPTION
**Pull request by valerianpereira**
`not` instead of `no` - typo

### CODE REVIEW

**Comment by valerianpereira** [comment]
I signed it

**Comment by valerianpereira** [comment]
I signed it

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ you are doing [manual node administration](#manual-node-administration), then yo
 capacity when adding a node.
 
 The Kubernetes scheduler ensures that there are enough resources for all the pods on a node.  It
-checks that the sum of the requests of containers on the node is no greater than the node capacity.  It
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md]: The original wording looks fine to me; it's a form of English that I'm accustomed to seeing.

Comment (sftim) [reply content/en/docs/concepts/architecture/nodes.md]: Some examples of “no greater than” in English: https://www.google.com/search?q="no+greater+than"

**Pull request closed without merging**